src/foundry/client/pixi/groups/environment.d.mts

```diff
@@ -15,6 +25,52 @@ declare global {
      */
     static override tearDownChildren: boolean;
 
-    override draw(): Promise<void>;
+    /**
+     * Colors exposed by the manager.
+     */
+    colors: Record<string, Color | undefined>;
+
+    /**
+     * Weights used by the manager to compute colors.
+     */
+    weights: Record<string, number | undefined>;
+
+    /**
+     * Get the darkness level of this scene.
+     */
+    get darknessLevel(): number;
+
+    protected override _draw(options: CanvasGroupMixin.DrawOptions): Promise<void>;
+
+    /**
+     * Initialize the scene environment options.
+     * @remarks `@fires PIXI.FederatedEvent type: "darknessChange" - event: {environmentData: {darknessLevel, priorDarknessLevel}}`
+     */
+    initialize(config: InexactPartial<EnvironmentCanvasGroup.InitializeConfig>): void;
+
+    /**
+     * @deprecated since v12, will be removed in v14
+     * @remarks `"EnvironmentCanvasGroup#darknessPenalty is deprecated without replacement. The darkness penalty is no longer applied on light and vision sources."`
+     */
+    get darknessPenalty(): 0;
```

Comment:
I won't make you change this because it's deprecated but just for what it's worth I try to generalize the types, i.e. make it `number` because depending on it being exactly `0` seems like a mistake for an end user since it should be changeable.